src/tests/functional/plugin/conformance/subgraphs_dumper/include/matchers/subgraph/repeat_pattern.hpp

```diff
@@ -5,31 +5,46 @@
 #pragma once
 
 #include <utility>
+
 #include "matchers/subgraph/subgraph.hpp"
-#include "matchers/single_op/single_op.hpp"
-#include "matchers/single_op/convolutions.hpp"
-#include "matchers/single_op/manager.hpp"
 
 namespace ov {
 namespace tools {
 namespace subgraph_dumper {
 
 class RepeatPatternExtractor final : public SubgraphExtractor {
+private:
+    using InputVector = std::vector<ov::Input<ov::Node>>;
+    using OutputVector = std::vector<ov::Output<ov::Node>>;
+
 public:
-    RepeatPatternExtractor() {
-        MatchersManager::MatchersMap matchers = {
-            { "generic_single_op", SingleOpMatcher::Ptr(new SingleOpMatcher) },
-            { "convolutions", ConvolutionsMatcher::Ptr(new ConvolutionsMatcher) },
-        };
-        manager.set_matchers(matchers);
-    }
-
-    std::list<ExtractedPattern> extract(const std::shared_ptr<ov::Model> &model,
-                                        bool is_extract_body = true,
-                                        bool is_copy_constants = true) override;
+    using PatternBorders = std::pair<InputVector, OutputVector>;
+
+    std::vector<std::vector<PatternBorders>>
```

Comment:
@jane-intel @slyalin Please look to the extended pattern API. Please approve if LGTU